cardano-chain-gen/test/Test/Cardano/Db/Mock/Unit/Babbage/Config/MigrateConsumedPruneTxOut.hs

```diff
@@ -291,15 +259,12 @@ pruneSameBlock =
     assertTxInCount dbSyncEnv 0
     assertEqQuery dbSyncEnv DB.queryTxOutConsumedCount 0 "Unexpected TxOutConsumedByTxId after rollback"
   where
-    cmdLineArgs =
-      initCommandLineArgs
-        { claConfigFilename = "test-db-sync-config-prune.json"
-        }
+    cmdLineArgs = initCommandLineArgs
     testLabel = "configPruneSameBlock"
 
 noPruneSameBlock :: IOManager -> [(Text, Text)] -> Assertion
 noPruneSameBlock =
-  withCustomConfig cmdLineArgs Nothing babbageConfigDir testLabel $ \interpreter mockServer dbSyncEnv -> do
+  withCustomConfig cmdLineArgs (Just configPruneForceTxIn) babbageConfigDir testLabel $ \interpreter mockServer dbSyncEnv -> do
```

Comment:
The file used was `test-db-sync-config-consumed.json`, so this should be consumed instead of prune